### PR TITLE
feat: new explore menu

### PR DESCRIPTION
**Pull request by rephus**
Closes: #3915
### Description:

[Screencast from 08-12-22 11:49:07.webm](https://user-images.githubusercontent.com/1983672/206428274-3447bb6b-a7b0-4742-bf33-d485725fab6e.webm)

With view permissions: 

Screenshot from 2022-12-08 11-44-08

### CODE REVIEW

**Comment by PriPatel** [comment]
Looks good @rephus !

File: packages/frontend/src/components/NavBar/ExploreMenu/index.tsx [hunk 1]
```diff
@@ -1,64 +1,170 @@
...
-                                text="Tables"
+                                title="Query from tables"
+                                description="Build queries from your tables to turn them into saved charts."
+                            />
+                        </ButtonWrapper>
+                        <ButtonWrapper
+                            onClick={() => {
+                                setIsOpen(false);
+                                history.push(
+                                    `/projects/${projectUuid}/sqlRunner`,
```
Comment (ZeRego) [inline packages/frontend/src/components/NavBar/ExploreMenu/index.tsx:90]: I think we are missing the SQL runner check we had before

```<Can
                            I="manage"
                            this={subject('SqlRunner', {
                                organizationUuid: user.data?.organizationUuid,
                                projectUuid,
                            })}
                        >```

File: packages/frontend/src/components/NavBar/ExploreMenu/index.tsx [hunk 2]
```diff
@@ -1,64 +1,170 @@
...
-                        <Spinner size={20} />
-                    </SpinnerWrapper>
-                ) : (
-                    <Menu>
-                        <NavLink to={`/projects/${projectUuid}/tables`}>
-                            <FirstItem
-                                roleStructure="menuitem"
+        <>
+            <Popover2
+                isOpen={isOpen}
```
Comment (ZeRego) [inline packages/frontend/src/components/NavBar/ExploreMenu/index.tsx:70]: At the moment the popup seems to be persistent. 


persistent popup

**Comment by PriPatel** [comment]
Hey @rephus - Here is the new copy as discussed:

**Query from tables**
Build queries and save them as charts

**Query using SQL runner**
Access your databases to run ad-hoc queries


This is so that every description spans 2 lines -  thank you!

**Review by ZeRego — APPROVED** [review]

**Pull request merged**